inc/thorvg.h

```diff
@@ -1339,7 +1339,17 @@ class TVG_EXPORT Initializer final
 /**
  * @class Saver
  *
- * @brief A class enabling saving a paint in a binary format.
+ * @brief A class exporting a paint in a specified file data to recover the paint scene later.
+ *
+ * ThorVG provides a feature for exporting & importing paint data, the Saver has a role to export it to a file.
+ * Basically, this feature is useful when you need to save the composed scene or image from a paint object and replay it later.
+ *
+ * The file format is decided by the extension name(i.e. "*.tvg") while the supported formats depend on the TVG packaging environment.
+ * If it doesn't support the file format, it will return the @c NonSuppport result by the save() method.
```

Comment:
When trying to save a file with an unknown extension or a non-supported format, the @c NonSuppport result is returned by the save() method.

Reply:
updated